samples/network/README.md

```diff
@@ -0,0 +1,63 @@
+# How to use Berkeley/Posix Socket APIs in WebAssembly
+
+**_Berkeley sockets_** usually means an API for Internet sockets and Unix domain
+sockets. A socket is an abstract representation of the local endpoint of a
+network communication path.
+
+Temporarily, WAMR supports a limit set of all well-known functions:
+`accept()`, `bind()`, `connect()`, `listen()`, `recv()`, `send()`, `shutdown()`
+and `socket()`. Users can call those functions in WebAssembly code directly.
+Those WebAssembly socket calls will be dispatch to imported
+functions and eventually will be implemented by host socket APIs.
+
+This document will introduce a way to support _Berkeley/Posix Socket APIs_ in
+WebAssembly code.
+
+## Patch the native code
+
+The first step is to add the header of the WAMR socket extension into the native
+source code.
+
+```c
+#ifdef __wasi__
+#include <wasi_socket_ext.h>
+#endif
+```
+
+`__wasi__` is a Marco defined by WASI. The host compiler will not enable it.
+
+## CMake files
+
+It is recommended that use CMake to build the project. Use [_wasi-sdk_](https://github.com/WebAssembly/wasi-sdk)
+as a toolchain to compile C/C++ to WebAssembly
+
+```bash
+$ CMAKE -DWASI_SDK_PREFIX=${WASI_SDK_DIR}
+      -DCMAKE_TOOLCHAIN_FILE=${WASI_TOOLCHAIN_FILE}
+      -DCMAKE_SYSROOT=${WASI_SYS_ROOT}
+      ..
+```
+
+Include an extension of socket support and link with it.
```

Comment:
Does it mean that in the cmake file, we need to include the following lines? If yes, how about changing to:
`And in the cmake file, include an extension ...`